src/global.css

```diff
@@ -53,8 +53,9 @@
   }
 
   .default-toggle {
-    @apply w-12 h-10 text-f-primary
-      hover:bg-secondary                                            
+    @apply w-12 h-10 text-primary border-0
```

Comment:
For what you override border with `border-0` here ?

I think if it's to remove border in the `.default-button`, you can move `border` value in `global.css` into `Button.tsx` file.
But as you want.

Reply:
That's a good option indeed!

Just updated!

Reply:
All good!